module/VuFind/src/VuFind/RecordDriver/SolrOverdrive.php

```diff
@@ -563,4 +575,41 @@ public function getRealTimeTitleHold()
         ];
         return $urlDetails;
     }
+
+    /**
+     * Return an array of associative URL arrays with one or more of the following
+     * keys:
+     *
+     * <li>
+     *   <ul>desc: URL description text to display (optional)</ul>
+     *   <ul>url: fully-formed URL (required if 'route' is absent)</ul>
+     *   <ul>route: VuFind route to build URL with (required if 'url' is absent)</ul>
+     *   <ul>routeParams: Parameters for route (optional)</ul>
+     *   <ul>queryString: Query params to append after building route (optional)</ul>
+     * </li>
+     *
+     * @return array
+     */
+    public function getURLs()
+    {
+        $permlink =  $this->getPermanentLink();
+        //todo translate
+        $desc = 'Overdrive Resource Page';
```

Comment:
Record drivers have access to the translator, so this TODO is not hard at all -- just make sure the string is in the language file, and change this to `$desc = $this->translate(...)`. Additionally, if you're hard-coding $desc, there's no need to do the `?: $permlink` in the next line, since $desc will never be false.